Library/Homebrew/test/emoji_spec.rb

```diff
@@ -4,8 +4,15 @@
   describe "#install_badge" do
     subject { described_class.install_badge }
 
-    it "returns 🍺 by default" do
-      expect(subject).to eq "🍺"
+    context "with emoji environment variables unset" do
+      before(:each) do
+        ENV.delete("HOMEBREW_NO_EMOJI")
+        ENV.delete("HOMEBREW_INSTALL_BADGE")
+      end
+
+      it "returns 🍺 by default" do
```

Comment:
Also add two spaces after the emoji here, as the terminal seems to need two for an emoji.